tests/Doctrine/Tests/ORM/Functional/PersistentCollectionTest.php

```diff
@@ -95,6 +95,28 @@ public function testMatchingDoesNotModifyTheGivenCriteria()
         $this->assertEmpty($criteria->getMaxResults());
         $this->assertEmpty($criteria->getOrderings());
     }
+
+    public function testPersistClearCollection()
+    {
+        $collectionHolder = new PersistentCollectionHolder();
+        $content = new PersistentCollectionContent('first element');
```

Comment:
Class constructor doesn't take any argument, is `first element` part of your test?